src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -14,24 +14,26 @@ public enum CallType
         Method = 1,
         Set = 8,
     }
-    public sealed partial class Collection : System.Collections.ICollection, System.Collections.IList
+    public sealed partial class Collection : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList
```

Comment:
@cston, @333fred - Renamed the file to match assembly name. Please review this change.